### ISSUE TITLE
Improperly formatted number 

### ISSUE THREAD

**Issue by brymonster** [amoose136/call_or_sms_contact#2]
All contacts seem to be improperly formatted numbers. impossible. lol
![screen shot 2017-02-24 at 9 49 32 pm](https://cloud.githubusercontent.com/assets/12586478/23328642/69243db2-fadb-11e6-89af-4aa264863257.png)
Also any way to choose which number under a contact to call? mobile, home, work..

**Comment by amoose136** [comment]
In an attempt to make it more robust I implemented number format checking a while back. It worked okay for US people but unfortunately because the phone number standards vary so much throughout the world and you might have international contacts in your phone book, it's basically pointless. I committed [a big fat falsehood.](https://github.com/googlei18n/libphonenumber/blob/master/FALSEHOODS.md) Going down the rest of the list, implementing a proper checker is really hard and doesn't solve much so for now I will remove it shortly. That should fix whatever I broke in the last update.  

As to adding a way of choosing which number to call, no there is no way to do this right now with the way Alfred file filters work. I'm trying to recode the entire workflow from scratch in python to do this but it's no small feat for technical reasons. This is only something I occasionally do in my spare time so don't expect it for another month or so.
